Server/src/services/vehicle.service.ts

```diff
@@ -128,7 +95,7 @@ export default class VehicleService {
 				return null
 			}
 
-			allVehiclesOnTrack.filter(async function (vehicle, index, vehicles) {
+			allVehiclesOnTrack.filter(async function(vehicle, index, vehicles) {
```

Comment:
Please apply the formatting rules and let prettier add the whitespace in front of the function keyword. On the other hand, there are multiple eslint violations because the function contains unused arguments that don't follow the defined pattern.

Reply:
The formatting rules are applied. The whitespaces in front of the function keyword is not defined by the current prettier-config. That means it has to be added into the IDE-settings manually. I think we might need to add some 'Style Guide' in the README in the Server directory for those sorts of issues. 

I will have a look at those eslint-violations.

Reply:
Formatting rules.. the bane of our existence. I will check my IDE for the specific setting :D